compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -538,33 +538,34 @@ timedSection targetDiffTime block = do
         throwError $ TimedSectionTookTooLong targetDiffTime actualDiffTime
     return value
 
-actionsToChoiceActions :: Set.Set V.Action -> Set.Set ExpectedChoiceAction
-actionsToChoiceActions acts = Set.map expectedChcAction acts
-    where expectedChcAction = \case
-            V.ACreate tcon -> Create (DAP.renderPretty tcon)
-            V.AExercise tcon choice -> Exercise (DAP.renderPretty tcon) (DAP.renderPretty choice)
-
-templateChoicesToProps :: V.TemplateChoices -> TemplateProp
-templateChoicesToProps tca = TemplateProp tName choicesInTpl
-    where tName = V.tplNameUnqual (V.template tca)
-          choicesInTpl = Set.fromList $ map (\ca -> ExpectedChoice (DAP.renderPretty $ V.choiceName ca) (V.choiceConsuming ca) (actionsToChoiceActions $ V.actions ca)) (V.choiceAndActions tca)
-
-graphTest :: LF.World -> LF.Package -> Set.Set TemplateProp -> ShakeTest ()
-graphTest wrld lfPkg expectedProps = do
-    let actual = Set.fromList $ map templateChoicesToProps tplPropsActual
-        tplPropsActual = concatMap (V.moduleAndTemplates wrld) (NM.toList $ LF.packageModules lfPkg)
-    unless (expectedProps == actual) $
-        throwError $ ExpectedTemplateProps expectedProps actual
-
-expectedTemplatePoperties :: D.NormalizedFilePath -> Set.Set TemplateProp -> ShakeTest ()
-expectedTemplatePoperties damlFilePath expectedProps = do
+expectedSubgraph :: V.SubGraph -> ExpectedSubGraph
```

Comment:
Your names are inconsistent: `expectedSubgraph` but `graphToExpectedGraph` instead of `expectedGraph` which also exists but does something different.

Reply:
Thanks for point that out. Corrected it